pages/static/tools.html

```diff
@@ -0,0 +1,178 @@
+                https://i0.wp.com/humantumoratlas.wpcomstaging.com/wp-content/uploads/2020/11/Screen-Shot-2020-11-12-at-7.20.39-AM.png?resize=1024%2C512&amp;ssl=1  1024w,
+                https://i0.wp.com/humantumoratlas.wpcomstaging.com/wp-content/uploads/2020/11/Screen-Shot-2020-11-12-at-7.20.39-AM.png?resize=300%2C150&amp;ssl=1    300w,
+                https://i0.wp.com/humantumoratlas.wpcomstaging.com/wp-content/uploads/2020/11/Screen-Shot-2020-11-12-at-7.20.39-AM.png?resize=768%2C384&amp;ssl=1    768w,
+                https://i0.wp.com/humantumoratlas.wpcomstaging.com/wp-content/uploads/2020/11/Screen-Shot-2020-11-12-at-7.20.39-AM.png?resize=1536%2C769&amp;ssl=1  1536w,
+                https://i0.wp.com/humantumoratlas.wpcomstaging.com/wp-content/uploads/2020/11/Screen-Shot-2020-11-12-at-7.20.39-AM.png?resize=2048%2C1025&amp;ssl=1 2048w,
+                https://i0.wp.com/humantumoratlas.wpcomstaging.com/wp-content/uploads/2020/11/Screen-Shot-2020-11-12-at-7.20.39-AM.png?resize=1000%2C500&amp;ssl=1  1000w,
+                https://i0.wp.com/humantumoratlas.wpcomstaging.com/wp-content/uploads/2020/11/Screen-Shot-2020-11-12-at-7.20.39-AM.png?resize=500%2C250&amp;ssl=1    500w,
+                https://i0.wp.com/humantumoratlas.wpcomstaging.com/wp-content/uploads/2020/11/Screen-Shot-2020-11-12-at-7.20.39-AM.png?resize=1200%2C600&amp;ssl=1  1200w,
+                https://i0.wp.com/humantumoratlas.wpcomstaging.com/wp-content/uploads/2020/11/Screen-Shot-2020-11-12-at-7.20.39-AM.png?resize=100%2C50&amp;ssl=1     100w,
+                https://i0.wp.com/humantumoratlas.wpcomstaging.com/wp-content/uploads/2020/11/Screen-Shot-2020-11-12-at-7.20.39-AM.png?resize=1484%2C743&amp;ssl=1  1484w,
+                https://i0.wp.com/humantumoratlas.wpcomstaging.com/wp-content/uploads/2020/11/Screen-Shot-2020-11-12-at-7.20.39-AM.png?resize=600%2C300&amp;ssl=1    600w
+            "
+            sizes="(max-width: 600px) 100vw, 600px"
+            data-recalc-dims="1"
+    /></a>
+</p>
+<p>
+    <a
+        href="https://www.cbioportal.org/patient?studyId=brca_hta9_htan_2022&caseId=HTA9_1"
+        >Explore the Clinicogenomic Profiling of HTAN OHSU Case HTA9_1 in
+        cBioPortal</a
+    >
+</p>
+<h3>Google BigQuery</h3>
+<p>
+    <a href="https://cloud.google.com/bigquery">Google BigQuery</a> is a
+    massively-parallel analytics engine ideal for working with tabular data.
+    HTAN single cell and metadata BigQuery tables are available via the CRDC ISB
+    Cancer Gateway in the Cloud
+</p>
+<p>
+    <a href="https://isb-cgc.appspot.com/bq_meta_search/"
+        ><img
+            width="800"
+            src="https://user-images.githubusercontent.com/2837859/179309894-52004856-b61a-44ad-ac68-66539902b779.png"
+    /></a>
+</p>
+<p>
+    <a href="https://isb-cgc.appspot.com/bq_meta_search/"
+        >Explore HTAN BigQuery tables in ISB-CGC</a
+    >. Included are computational notebooks illustrating how to query and work
+    with HTAN data.
```

Comment:
Add text: "To explore HTAN BigQuery tables in ISB-CGC, select **Launch** under `BigQuery Table Search` and **HTAN** from the `Program` dropdown. To access HTAN R and Python notebooks, select **GitHub** under `Notebooks` and navigate to **HTAN** folder."